src/crimson/os/seastore/seastore.cc

```diff
@@ -1507,6 +1531,29 @@ SeaStore::Shard::_write(
     });
 }
 
+SeaStore::Shard::tm_ret
+SeaStore::Shard::_clone(
+  internal_context_t &ctx,
+  OnodeRef &onode,
+  OnodeRef &d_onode)
+{
+  LOG_PREFIX(SeaStore::_write);
+  DEBUGT("onode={} d_onode={}", *ctx.transaction, *onode, *d_onode);
+  return seastar::do_with(
+    ObjectDataHandler(max_object_size),
+    [this, &ctx, &onode, &d_onode](auto &objHandler) {
+    auto &object_size = onode->get_layout().size;
+    auto &d_object_size = d_onode->get_mutable_layout(*ctx.transaction).size;
```

Comment:
I think only omap may need to be considered, which I planned to do in follow-up PRs when the time is right. Right now, for RBD scenarios, this may not be necessary. What do you think?

Reply:
See https://github.com/ceph/ceph/blob/main/src/os/Transaction.h#L961-L971

Document the missing parts with TODO if won't be included in this PR.

Reply:
> See https://github.com/ceph/ceph/blob/main/src/os/Transaction.h#L961-L971
> 
> Document the missing parts with TODO if won't be included in this PR.

Ah, yes.